docs/reference/mapping/types/boolean.asciidoc

```diff
@@ -31,7 +31,7 @@ PUT my-index-000001
   }
 }
 
-POST my-index-000001/_doc/1
+POST my-index-000001/_doc/1?refresh
```

Comment:
```suggestion
POST my-index-000001/_doc/1
```

This may be an outdated best practice, but I thought that we preferred `?refresh=wait_for`. I'd also move this into the `// TEST` comment. This is unlikely to affect someone copy/pasting the example.

Reply:
`?refresh=wait_for` is pretty niche - I wouldn't want it in docs because it'll slow them down needlessly. I think users probably *should* see the `?refresh` because if they copy and paste the whole thing and run it all at once then it'll fail without refresh.

Reply:
Makes sense. I'm good with keeping those in and updating my mental database!